docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc

```diff
@@ -0,0 +1,91 @@
+
+= Cortex Time Series Plugin
+
+The Time Series Storage Plugin for Cortex helps {page-component-title} to persist aggregated flow data to Cortex.
+The plugin uses the xref:deployment:time-series-storage/timeseries/ts-integration-layer.adoc[time series storage integration layer] to convert metrics to a Prometheus model and delegate writes and reads to https://cortexmetrics.io/[Cortex].
+The plugin also makes it possible for https://docs.opennms.com/helm/latest/index.html[OpenNMS Helm] to visualize flow graphs from the Cortex instance.
```

Comment:
Let's be more specific about which part of Helm. It's the Helm Performance Data Source Plugin.

Reply:
>visualize flow graphs from the Cortex instance.

>It's the Helm Performance Data Source

Is it?

Reply:
I'll update the Helm sentence to be less project based, and more user based. Users can visualize flow graphs from Cortex in Helm through the performance data source. Something like that.

Reply:
@jeffgdotorg is there additional configuration required in Helm to make things work with Cortex? If so, we need to document it.